styles.css

```diff
@@ -0,0 +1,87 @@
+@import url('https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;700&family=Poppins&display=swap');
+
+* {
+  box-sizing: border-box;
+  max-width: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.nav-bar {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 10px;
+  padding: 12px;
+}
+
+.nav-bar a {
+  font-family: 'Poppins', sans-serif;
```

Comment:
## Optional Changes

- [ ] Good job including the font family required to be used in this project. I will suggest you move this `font-family: 'Poppins', sans-serif;` to the body styles of the entire web page that way you don't have to repeat calling it on all texts in the page. For as long as the site requires one font family, it's best you place that style on the body style so all texts, be it `p` `hx` or `a` tags will have that style.

**HINT**
- Add this style in line 9 or 10 of your CSS styles just after the default styles
```mdx
body{
  font-family: 'Poppins', sans-serif;
}
```
Remember DRY (Don't Repeat Yourself) principle. 👍